test/utils/isCloseTo.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import assert from 'assert';
+
+/**
+ * Returns if two numbers are within a maxDelta of each other
+ * @return Bool
+ */
+export function isCloseTo(actual, expected, maxDelta) {
```

Comment:
Nit: I would rename this to `assertIsCloseTo()` to match similar things we're already doing in the [LCP tests](https://github.com/GoogleChrome/web-vitals/blob/03388e81a28dafbef462399cdaa2a34a80fb2369/test/e2e/onLCP-test.js#L802).